src/main/java/jp/co/future/uroborosql/context/SqlContextImpl.java

```diff
@@ -66,7 +66,12 @@ public boolean contains(final Object o) {
 
 	/** where句の直後にくるANDやORを除外するための正規表現 */
 	protected static final Pattern WHERE_CLAUSE_PATTERN = Pattern
-			.compile("(?i)(WHERE(\\s+(/\\*.*\\*/|--.*)+)*\\s+)(AND\\s+|OR\\s+)");
+			.compile("(?i)(WHERE\\s+(--.*|/\\*.*\\*/\\s*)*\\s*)(AND\\s+|OR\\s+)");
+
+	/** 各句の最初に現れるカンマを除去するための正規表現 */
+	protected static final Pattern REMOVE_FIRST_COMMA_PATTERN = Pattern.compile("(?i)(((SELECT|ORDER BY|GROUP BY|SET)\\s+|\\(\\s*)(--.*|/\\*.*\\*/\\s*)*\\s*)(,)");
```

Comment:
@HidekiSugimoto189 Is not `ORDER\\s+BY` better instead of `ORDER BY`? `GROUP BY` is similar.

Reply:
I see. Fixed.